packages/reporter/src/commands/commands.scss

```diff
@@ -170,11 +105,15 @@
 
   .command-pin-target.command-group {
     border-left: 1px dotted $gray-600;
+    border-image-slice: 0 0 0 1;
+    border-image-source: repeating-linear-gradient(0deg, transparent, $gray-600, $gray-600 4px);
```

Comment:
Would you be open to leverage a [mixin](https://sass-lang.com/documentation/at-rules/mixin) that takes variables for the gradient?

Reply:
done: [`ecac805` (#22857)](https://github.com/cypress-io/cypress/pull/22857/commits/ecac80537b6bd1ba42758df2b927e0c2bf3a15f6#diff-c7e26e15bb4828e49b88a9173bb9596756a2335462b1242305741eb076d98f32R1)